src/com/owncloud/android/ui/fragment/ExtendedListFragment.java

```diff
@@ -327,6 +357,23 @@ public void setSwipeEnabled(boolean enabled) {
     }
 
     /**
+     * Disables FAB.
```

Comment:
The method allows both disabling or enabling. Removing the line the rest of the comment seemed to describe perfectly the method. I actually thought this was a forgotten line of a previous version of the method.

Of course, this is unimportant, sorry  for being so punctilious :$